src/strategies/safe-vested/index.ts

```diff
@@ -0,0 +1,85 @@
+import fetch from 'cross-fetch';
+import { formatUnits, parseUnits } from '@ethersproject/units';
+
+import { Multicaller } from '../../utils';
+
+export const author = 'dasanra';
+export const version = '0.1.0';
+
+const abi = [
+  'function vestings(bytes32) view returns (address account, uint8 curveType, bool managed, uint16 durationWeeks, uint64 startDate, uint128 amount, uint128 amountClaimed, uint64 pausingDate, bool cancelled)'
```

Comment:
Should we link to the vesting contract in the safe token repo?

Reply:
@rmeissner Done 👍